test/augmentation/test_torchaudio.py

```diff
@@ -36,33 +43,66 @@ def test_speed_does_not_change_num_samples(audio):
         augmented_audio = augment_fn(audio, sampling_rate=SAMPLING_RATE)
         assert augmented_audio.shape == audio.shape
         assert augmented_audio != audio
+        
+        
+def test_vol_does_not_change_num_samples(audio):
+    augment_fn = SoxEffectTransform(effects=vol(SAMPLING_RATE))
+    for _ in range(10):
+        augmented_audio = augment_fn(audio, sampling_rate=SAMPLING_RATE)
+        assert augmented_audio.shape == audio.shape
+        assert augmented_audio != audio
 
 
 def test_speed(audio):
     speed = Speed(factor=1.1)
     perturbed = speed(audio, SAMPLING_RATE)
     assert perturbed.shape == (1, 14545)
+    
+
+@pytest.mark.parametrize('scale', [0.125, 1., 2.])
+def test_vol(audio_vol, scale):
+    vol = Vol(factor=scale)
+    
+    audio_perturbed = vol(audio_vol, SAMPLING_RATE)
+    
+    assert audio_perturbed.shape == audio_vol.shape
+    assert_array_almost_equal(audio_perturbed, scale * audio_vol)
 
 
 def test_deserialize_transform(audio):
     speed = AudioTransform.from_dict({'name': 'Speed', 'kwargs': {'factor': 1.1}})
-    perturbed = speed(audio, SAMPLING_RATE)
-    assert perturbed.shape == (1, 14545)
```

Comment:
These changes would ideally be introduced in a copy of this unit test (my principle is one logical assertion == one unit test, to avoid tests that check independent functionalities because they are confusing to readers and harder to debug)